CHANGELOG.md

```diff
@@ -1,4 +1,23 @@
 # Changelog
+
+## 1.2.0
```

Comment:
we need to be careful about the order of merging now. 

this is 1.2.0 and the old vor removal is 2.0.0. this will have to come first.